### PR TITLE
Added ARG query for "Design a monitoring strategy to send metrics and logs to an Log Analytics workspace" in azure_arc_checklist.en.json

### PR DESCRIPTION
**Pull request by denver-reed**
## Description
Please provide a detailed description of your changes.
ARG query added for checking to see if Azure Arc machines have the Azure Monitoring agent extension installed. Will return non-compliant if agent is missing.

## Related Issue
Link to any related issues or discussions here. This helps reviewers understand the context and the need for your changes.

## Checklist

- [x] I've tested my changes to ensure they are ready for review.
- [x] I've read the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md) guide.
- [ ] I've updated the documentation (if applicable).
- [x] Resource Graph queries have been included (and tested) for recommendations where ever possible[^note].
- [ ] Resource Graph queries have NOT been included (please explain below).

## Additional Information
Is there any additional context, screenshots, or considerations that might help in the review process? Please include them here.

## Reviewer Notes
Is there a specific area you’d like feedback on? Please highlight it here. We're here to help and learn together! 💡

[^note]:
    Details on how to add Azure Resource Graph queries to recommendations can be found [here](../blob/main/CONTRIBUTING.md#1-adding-or-modifying-resource-graph-queries).

### CODE REVIEW

**Comment by jodyford-msft** [comment]
Please add line item ID and checklist filename to the title of your PR

**Comment by denver-reed** [comment]
> Please add line item ID and checklist filename to the title of your PR

Confirmed file name is in the title. Unfortunately, for this checklist, there are no line item IDs.

File: checklists/azure_arc_checklist.en.json [hunk 1]
```diff
@@ -212,6 +212,7 @@
             "text": "Design a monitoring strategy to send metrics and logs to an Log Analytics workspace",
             "guid": "74d1102c-ac6a-4ae0-8e6a-84de5df47d2d",
             "severity": "Medium",
+            "graph": "resources | where type =~ 'microsoft.hybridcompute/machines' | project id, arcMachineName = name | join kind=leftouter (resources | where type == 'microsoft.hybridcompute/machines/extensions' | where name contains 'azuremonitor' | extend baseIdParts = split(id, '/extensions/') | extend extensionMachineId = tostring(baseIdParts[0]) | project extensionMachineId, extensionId = id) on $left.id == $right.extensionMachineId | extend compliant = iif(extensionMachineId == '', 'No', 'Yes') | project id, compliant",
```
Comment (erjosito) [inline checklists/azure_arc_checklist.en.json:215]: * `compliant` should be a boolean. You could do something like this: `| extend compliant = isnotempty(extensionMachineId)` (see https://learn.microsoft.com/en-us/kusto/query/isnotempty-function?view=microsoft-fabric for more info on the `isnotempty` function)

**Review by erjosito — CHANGES_REQUESTED** [review]

File: checklists/azure_arc_checklist.en.json (hunk 1 above)
Comment (denver-reed) [reply checklists/azure_arc_checklist.en.json]: I believe I made the change you requested. Thanks @erjosito!

**Review by erjosito — APPROVED** [review]

**Comment by erjosito** [comment]
@seenu433 It looks like there are some conflicts?

**Pull request merged**